lib/commands/fun/RollCommand.js

```diff
@@ -31,38 +31,26 @@ module.exports = class RollCommand extends Command {
         let iconUrl = 'https://i.imgur.com/aw0Oa4p.png';
         let rangeStartOrDice = args.get('rangeStartOrDice');
         let rangeEnd = args.get('rangeEnd');
+         
+        // check if first arg matches a 'd6' style pattern
+        if (rangeStartOrDice && rangeStartOrDice.match(/d\d+/i) && !rangeEnd) {
+            let diceType = rangeStartOrDice.match(/d\d+/i);
+            let diceValue = parseInt(diceTypegit [0].slice(1));

```

Comment:
Typo here

Reply:
Additionally, could store the regex match above, check if it worked in the if statement and use the result in the block.